actioncable/lib/action_cable/process/logging.rb

```diff
@@ -1,7 +1,8 @@
 require 'action_cable/server'
-require 'eventmachine'
 
-EM.error_handler do |e|
-  puts "Error raised inside the event loop: #{e.message}"
-  puts e.backtrace.join("\n")
+if defined?(::EventMachine)
+  EventMachine.error_handler do |e|
+    puts "Error raised inside the event loop: #{e.message}"
+    puts e.backtrace.join("\n")
+  end
```

Comment:
Yeah. I think I'll just kill it; we're not doing "real work" in the EM loop any more, so it's unlikely to be interesting.